server/mdm/nanomdm/storage/mysql/mysql.go

```diff
@@ -274,3 +298,24 @@ func (s *MySQLStorage) updateLastSeen(r *mdm.Request) (err error) {
 	}
 	return
 }
+
+func (s *MySQLStorage) updateLastSeenBatch(ctx context.Context, ids []string) {
+	if len(ids) == 0 {
+		return
+	}
+
+	stmt, args, err := sqlx.In(`UPDATE nano_enrollments SET last_seen_at = CURRENT_TIMESTAMP WHERE id IN (?)`, ids)
+	if err != nil {
+		s.logger.Info("msg", "error building nano_enrollments.last_seen_at sql", "err", err)
+		return
+	}
+
+	res, err := s.db.ExecContext(ctx, stmt, args...)
+	if err != nil {
+		s.logger.Info("msg", "error batch updating nano_enrollments.last_seen_at", "err", err)
+	} else {
+		// TODO(mna): temporary, to confirm updates in load tests
```

Comment:
TODO: remove this logging, was just to confirm use of new behavior in load test.